src/app/components/HumidityCard.tsx

```diff
@@ -0,0 +1,24 @@
+import { roundNumber } from "../util/math";
+import WeatherCard from "./WeatherCard";
+import { WiHumidity } from "rocketicons/wi";
+
+interface HumidityCardProps {
```

Comment:
```suggestion
export interface HumidityCardProps {
```
export props interface